api/resources_portal/models/attachment.py

```diff
@@ -53,3 +66,8 @@ def download_url(self):
             return "https://s3.amazonaws.com/" + self.s3_bucket + "/" + self.s3_key
         else:
             return None
+
+    def clean(self):
+        super(Attachment, self).clean()
+        if not (self.owned_by_org or self.owned_by_user):
+            raise ValidationError("Either owned_by_user or owned_by_org must be set on Attachment")
```

Comment:
Hmm, I think it might make sense to always have `owned_by_user` be the user that uploaded the file, even if they did so from the organization side of things. Then this method won't do much any more.

Reply:
This error now seems unnecessary.